packages/@aws-cdk/aws-apigateway/lib/vpc-link.ts

```diff
@@ -1,7 +1,18 @@
 import * as elbv2 from '@aws-cdk/aws-elasticloadbalancingv2';
-import { Construct, Lazy, Resource } from '@aws-cdk/core';
+import { Construct, IResource, Lazy, Resource } from '@aws-cdk/core';
 import { CfnVpcLink } from './apigateway.generated';
 
+/**
+ * A VPC Link.
+ */
+export interface IVpcLink extends IResource {
+  /**
+   * The VPC Link identifier.
```

Comment:
Use the same documentation text as in `VpcLink`

```suggestion
   * Physical ID of the VpcLink resource
```

Reply:
Please address this comment